### DISCUSSION TITLE
Face rendering?

### DISCUSSION THREAD

**Discussion by Le0X8** [vladmandic/human#340] [Q&A] [answered]
Is there a way to render a face from its embedding?
Would be fine :)

**Comment by vladmandic** [comment, marked as answer]
embedding can mean different things, this is not stable diffusion :)
even in case of stable diffusion, it would still need a large pretrained model to base embedding on.
so in short, no, there is no way.

**Comment by nicoeiris11** [comment]
In my case I needed to visualize the detected face to debug subsequent models' input and used the following:

- `userConfig.face.detector.return = true`
- https://stackoverflow.com/questions/64483632/convert-prediction-tensor-to-an-image

Don't know if this is exactly what you need but just in case I share how I did it.
